docs/topics/images.md

```diff
@@ -414,7 +416,33 @@ Note that this will have no effect on PNG or GIF files. If you want all images t
 {% image page.photo width-400 format-webp webpquality-50 %}
 ```
 
-### Generating image renditions in Python
+## Generating image renditions in Python
 
 All of the image transformations mentioned above can also be used directly in Python code.
 See [](image_renditions).
+
+(svg_images)=
+
+## SVG images
+
+Wagtail supports the use of Scalable Vector Graphics alongside raster images. To allow Wagtail users to upload and use SVG images, set `WAGTAILIMAGES_ALLOW_SVG` to `True`.
```

Comment:
Can you add this to the settings reference too please?

Reply:
I've removed this subsequent to #9860 - see https://github.com/wagtail/wagtail/pull/9842/commits/5812c0763d50320924ef239fdc6b19413f49d8b3